x/profiles/keeper/keeper.go

```diff
@@ -71,8 +70,7 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) StoreProfile(ctx sdk.Context, profile *types.Profile) error {
 	addr := k.GetAddressFromDTag(ctx, profile.DTag)
 	if addr != "" && addr != profile.GetAddress().String() {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest,
-			"a profile with DTag %s has already been created", profile.DTag)
+		return sdkerrors.Wrapf(types.ErrProfileAlreadyCreated, profile.DTag)
```

Comment:
If we are not using any argument, we can simply use `Wrap`

```suggestion
		return sdkerrors.Wrap(types.ErrProfileAlreadyCreated, profile.DTag)
```